requirements.txt

```diff
@@ -1,5 +1,6 @@
 apypie
 pytest-base-url
+pytest-html
```

Comment:
Why is this needed? Is it for https://github.com/theforeman/forklift/pull/1262?

Reply:
sorta. pytest-selenium does require pytest-html anyways (and without using the html report, no logs are fetched :roll_eyes:), but it felt weird to call `--html` without an explicit dependency